### PR TITLE
SPDX Defects Team meeting - 16th March 2022

### PR DESCRIPTION
**Pull request by rnjudge**
Signed-off-by: Rose Judge <rjudge@vmware.com>

### CODE REVIEW

File: defects/2022-03-16.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+* Jeff Can we add Option 4 to capture the proposed options?
+* Reference types - comments more flexible than identifiers. 
+* Thomas: This is an initial release and not the perfect solution. A more elegant solution may come later in SPDX 3.0
+* Dick: Need to be able to publish SBOMs in August to meet US Executive Order
+* Thomas: SPDX 2.3 deadline is early April to meet release date
+* Thomas: could use comment field to convey more information in SPDX 2.3
+* Thomas: Option 2 is good enough with maybe some consensus required on the categories. Option 1 is more restricted (and is included in Option 2 via web category)
+* Thomas: Start with simple set initially 
+* Dick: Need to define semantics for each category in Option 2
+* Jeff: Option 2 options for single use case with regard to supported reference types
```
Comment (jeff-schutt) [inline defects/2022-03-16.md:55]:
```suggestion
* Jeff: supported reference types in Option 2 only address one of the discussed use cases and standards
```

File: defects/2022-03-16.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+### Dick Brook's proposed practices document for SBOM relating to patches
+* Thomas: could also use PACKAGE_MANAGER ExternalRef type instead of just SECURITY. Allows consumers of the SBOM to look up vulnerabilities from the identifiers.
+* Discussion of SPDX root documents and where external reference references. The various external references could be at the root level or the package level depending on the context.
+* Karsten: External reference is used to query vulnerability database to identfy the vulnerabilities
+* Karsten: May need to be able to reference multiple CPEs
+* Thomas: that can be easily accomodated including vendor change
+* Dick: Option 3 is reasonable approach to go forward with. Can also use option 1 and 2 as necessary
+* Anthony: Can we have some examples to see how the proposal would look?
+* Thomas will look into it
+* Jeff Can we add Option 4 to capture the proposed options?
```
Comment (jeff-schutt) [inline defects/2022-03-16.md:46]:
```suggestion
* Jeff Can we add Option 4 to capture the proposed options?
* Jeff shared a table comparing the currently proposed options and additional option 4
```

File: defects/2022-03-16.md (hunk 1 above)
Comment (rnjudge) [reply defects/2022-03-16.md:55]: Updated

File: defects/2022-03-16.md (hunk 2 above)
Comment (rnjudge) [reply defects/2022-03-16.md:46]: Updated

**Review by tsteenbe — APPROVED** [review]

**Pull request merged**